examples/hello_world/hello_world-lock.ts

```diff
@@ -35,22 +36,25 @@ async function readValidator(): Promise<SpendingValidator> {
 const publicKeyHash = lucid.utils
   .getAddressDetails(await lucid.wallet.address())
   .paymentCredential
-  .hash;
+  ?.hash;
 
-const datum = Data.to(new Constr(0, [ publicKeyHash ]));
+const datum = Data.to(new Constr(0, [publicKeyHash]));
 
-const txLock = await lock(1000000, { into: validator, owner: datum });
+const txHash = await lock(BigInt(1000000), { into: validator, owner: datum });
 
-await lucid.awaitTx(txLock);
+await lucid.awaitTx(txHash);
 
-console.log(`1 ADA locked into the contract
-    Tx ID: ${txLock}
+console.log(`1 ADA locked into the contract at:
+    Tx Hash: ${txHash}
```

Comment:
I'll revert that one. We want to stick to `Tx ID`

Reply:
Hmm. I reckon that this is to be consistent with Lucid's naming. Which then I'd agree is a good thing.